e2e/test/scenarios/collections/personal-collections.cy.spec.js

```diff
@@ -46,6 +46,46 @@ describe("personal collections", () => {
       });
     });
 
+    it("should see a link to other users' personal collections only if there are other users", () => {
+      cy.intercept("GET", "/api/user", {
+        statusCode: 200,
+        body: {
+          data: [],
+          total: 1,
+          limit: 0,
+          offset: 0,
+        },
+      }).as("getUsers");
+
+      cy.visit("/");
+
+      cy.wait("@getUsers");
+
+      // Wait for page to re-render after user data is requested, to make the
+      // should("not.exist") test legit
+      cy.wait(200);
```

Comment:
Is there a way to avoid this? Maybe we could find something we know is on the rendered page first?